### PR TITLE
Add GitHub Action for unit tests

### PR DESCRIPTION
**Pull request by ralbayaty**
## Description of the change

- Moved the issue and PR templates to `.github` directory.
- Added a workflow to run unit tests

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 ## Related issues
 
-> Fix [#1]()
+> Fix [#<Issue number here>]()
```
Comment (newhouse) [inline .github/pull_request_template.md:12]: 😄

**Review by newhouse — APPROVED** [review]
NOICE.
What's the idea of the `unit-test.yml` vs what we have today?

**Comment by ralbayaty** [comment]
> What's the idea of the `unit-test.yml` vs what we have today?

I don't understand the question, specifically "what we have today".

**Comment by newhouse** [comment]
> > What's the idea of the `unit-test.yml` vs what we have today?
> 
> I don't understand the question, specifically "what we have today".

I'm just wondering what this thing does for us? Weren't we already running Unit Tests as part of our Actions on every commit/push?

**Comment by ralbayaty** [comment]
> > > What's the idea of the `unit-test.yml` vs what we have today?
> > 
> > 
> > I don't understand the question, specifically "what we have today".
> 
> I'm just wondering what this thing does for us? Weren't we already running Unit Tests as part of our Actions on every commit/push?

Ah I understand now. No, this repo was not running any GitHub workflows previously. Now this runs the tests in PRs and in master.

**Pull request merged**